src/components/projects/project-card.svelte

```diff
@@ -14,10 +14,12 @@
   export let start_date;
   export let end_date;
   export let moderated = false;
+
+  const image_placeholder = '/images/create-product/placeholder.svg';
 </script>
 
 <Card classname="with-image">
-  <img src="{API_HOST}/file/{image_id}" alt="" class="image" />
+  <img src={image_id == null ? image_placeholder :`${API_HOST}/file/${image_id}`} alt="" class="image" />
```

Comment:
Spaces after the colon, gentleman

Reply:
and that's what happens when you try to rush the PR 5 mins before the lab